src/util/waitForElement.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Waits until Element will appear
+ *
+ * @api private
+ * @method _waitForElement
+ * @param {string} elSelector Selector to locate Element
+ * @param {() => void} callback Callback to be called after Element appearance
+ */
+export default function _waitForElement(elSelector, callback) {
+  if (document.querySelector(elSelector) !== null) {
+    callback();
+    return;
+  }
+
+  if (typeof MutationObserver !== "undefined") {
+    /* jshint ignore:start */
```

Comment:
Why are we suppressing jshint errors? Happy to help if there are any issues with our jshint setup

Reply:
Yep, it could be deleted. Added MutationObserver to globals in jshint config and removed jshint ignore (33e560b3f576d99f9d134cc6a317e9945c6f0082)